test/lib/code_corps_web/views/conversation_view_test.exs

```diff
@@ -14,6 +13,52 @@ defmodule CodeCorpsWeb.ConversationViewTest do
         data: conversation |> Repo.preload(:conversation_parts)
       )
 
+    expected_json = %{
+      "data" => %{
+        "id" => conversation.id |> Integer.to_string,
+        "type" => "conversation",
+        "attributes" => %{
+          "read-at" => conversation.read_at,
+          "status" => conversation.status,
+          "inserted-at" => conversation.inserted_at,
+          "updated-at" => conversation.updated_at
```

Comment:
Make sure to try to alpha order these so it's quicker to scan.

Reply:
Today is the day I start alpha ordering always :laughing: